pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -37,6 +37,13 @@ var (
 	_ inject.Client             = (*ControlPlaneMutator)(nil)
 	_ admission.DecoderInjector = (*ControlPlaneMutator)(nil)
 
+	featureDisabled = false
```

Comment:
Should this be `:=`?  Is the address of this used elsewhere?

Reply:
No, you can't use `:=` with `var`.